bootstrap/roles/ocp4-install-quay/templates/quay-subscription.yaml.j2

```diff
@@ -0,0 +1,11 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: quay-operator
+  namespace: openshift-operators
+spec:
+  channel: stable-3.7
```

Comment:
could be worth it to use {{ quay-operator-version }} instead of hardcoding the version of Quay? it's only valid for stable-3.7?